### ISSUE TITLE
Do not prompt the user to see if they want to download the source text update when deselecting a source text

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord-dev/ts-android#2006]
Create a project and select a source text for which an update is available
The source moves to the Selected group and shows the update arrow
Click on the check box to deselect the source text
The app prompts the user to specify whether or not they want to download the update. We should assume that since they are deselecting the source they probably do not want to download it.

**Comment by PhotoNomad0** [comment]
@cckozie @neutrinog - the checkbox on the line is not actually a button.  We currently only detect pressing on the line.  Therefore we are making no assumptions about whether they want to deselect or download.  We ask if they want to download the update, and if they select no we presume they just wanted to deselect.

We could make the checkboxes separately clickable, but then the download icon is right next to it.  That could cause confusion.  Changing this might be a bit of work for something that is an annoyance.  We should also increase the size of the button to make sure it is easy to press on small screens (less than 7")

Or we could presume that if the item was selected and they click on it, then they want to deselect it. Then the next time they click on it they would get the prompt to download.  But it would slow them down trying to find the item in the list again.

I would suggest leaving this for a later release if necessary, since nothing is actually broken.  At most it is a minor annoyance.

**Comment by cckozie** [comment]
I agree based on that explanation that it doesn't make sense to do now. I'm moving it to the backlog.
